ddtrace/internal/telemetry/writer.py

```diff
@@ -55,6 +55,7 @@
 from .constants import TELEMETRY_INJECT_WAS_ATTEMPTED
 from .constants import TELEMETRY_LIB_INJECTION_FORCED
 from .constants import TELEMETRY_LIB_WAS_INJECTED
+from .constants import TELEMETRY_LOG_LEVEL  # noqa:F401
```

Comment:
Do we need no noqa here? Can we format this file using 'hatch run fmt:lint'

Reply:
I had to `noqa` here because it's only used in the type hint comment and would show up as an unused import.

We don't have a `fmt` hatch env, we do have a `lint` env with a `fmt` script, but it doesn't touch this.